src/HtmlHighlighter.php

```diff
@@ -56,6 +56,7 @@ public function highlightToken(int $type, string $value): string
         return '<span ' . $attributes . '>' . $value . '</span>';
     }
 
+    /** @param Token::TOKEN_TYPE_* $type */
```

Comment:
Maybe those improvements should happen in a separate PR.

Reply:
Sure, moved to #125.